CONTRIBUTING.md

```diff
@@ -10,35 +10,41 @@ Note: Due to a [limitation in textual](https://github.com/willmcgugan/textual/is
 
 2. Clone the repo and enter the `wtpython` folder.
 
-```
+```sh
 git clone https://github.com/<your-name>/wtpython.git
 cd wtpython
 ```
 
 3. Create and activate a virtual environment.
 
-```
+```sh
 python -m venv .venv --prompt template
 source .venv/bin/activate
 ```
 
-4. Upgrade pip and install [Flit](https://flit.readthedocs.io/en/latest/).
+4. Upgrade pip and install [Poetry](https://python-poetry.org/docs/#installation)
 
-```
-python -m pip install --upgrade pip flit
+```sh
```

Comment:
I might not tell them how to do it.

It didn't work for me when I tried the installs below, but the new install worked: https://python-poetry.org/docs/master/ which uses `install-poetry.py` install of `get-poetry.py`.

I think we should switch to that one, then I'll approve this for merge. Please also add a note about not using the old installer and other locations of poetry might not work well as they are outdated.